semeio/jobs/scripts/design2params.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+#
+import argparse
+import logging
+import os
+
+from semeio import valid_file
+from semeio.jobs.design2params import design2params
+
+description = """
+Reads a design matrix in XLSX-format and:
+ * Converts specified worksheet to txt-format and puts designmatrix.txt in RUNPATH
+ * Fetches values belonging to given ERT realization and *appends*
+   key-value pairs to parameters.txt
+ * Creates designparameters.txt which only contains variables taken from the design matrix, this
+   can be given to CUSTOM_KW
+Requires a matrix with column header as strings in topmost row in Excel.
+Row 2 in Excel must then correspond with the data you want for 'realization 0'
+Column 1 in Excel should contain the realization number, and will be ignored by this script.
+You must run this script as a FORWARD_MODEL from your ERT config, before
+you run DESIGN_KW.
+
+Usage:
+ $ design2params.py <realization-id> <xls-filename> <worksheet-name> [<defaultssheetname>]
+defaultssheetname is an optional argument, you may choose not to provide it (just give an empty string)
+Haavard Berland, Oseberg Petec, havb@statoil.com, April-June 2015
+"""
+
+
+def main():
+    parser = argparse.ArgumentParser(description=description)
+
+    parser.add_argument(
+        "realization", type=int, help="Which realization",
+    )
+
+    parser.add_argument(
+        "xlsfilename", type=valid_file, help="Path to design-matrix (xlsx file)",
+    )
+
+    parser.add_argument(
+        "designsheetname", type=str, help="Design sheet name",
+    )
+
+    parser.add_argument(
+        "defaultssheetname",
+        type=str,
+        nargs="?",
+        help="Defaults sheet name",
+        default=None,
+    )
+
+    parser.add_argument(
+        "--log-level", required=False, default="WARNING", type=logging.getLevelName,
+    )
+
+    args = parser.parse_args()
+    design2params.run(
+        realization=args.realization,
+        xlsfilename=args.xlsfilename,
+        designsheetname=args.designsheetname,
+        defaultssheetname=args.defaultssheetname,
+        log_level=args.log_level,
+    )
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Missing `args`, should perhaps have a test on the entry point?

Reply:
ah, didn't see your suggestion before reimplementing - but i think i've done almost what you suggested